enkit/version/version.go

```diff
@@ -36,6 +36,7 @@ func (r *Root) Run(cmd *cobra.Command, args []string) error {
 	}
 	fmt.Printf("Built from branch: %s\n", stamp.GitBranch)
 	fmt.Printf("Builder: %s\n", stamp.BuildUser)
+	fmt.Printf("Built at: %s\n", stamp.BuildTimestamp())
```

Comment:
For human consumption? yes. Not for script consumption, right?

Reply:
Yes, human consumption